### DISCUSSION TITLE
how to enable - ngx_http_geoip_module

### DISCUSSION THREAD

**Discussion by redimongo** [NginxProxyManager/nginx-proxy-manager#3126] [General]
Just wondering is it possible to enable ngx_http_geoip_module while using the docker image? If so how would we do it please.

**Comment by lastsamurai26** [comment]
You can try this one 
https://decatec.de/home-server/nginx-besucher-mittels-geoip2-nach-laendern-blockieren-geoblocking/

it is a german site but it works for me

**Comment by wpresident** [comment]
@redimongo working solution?
btw you can grab mmdb file https://github.com/P3TERX/GeoLite.mmdb without registration
